packages/edit-site/src/components/sidebar-navigation-screen-templates/index.js

```diff
@@ -40,7 +40,7 @@ const config = {
 			title: __( 'Library' ),
 			loading: __( 'Loading library' ),
 			notFound: __( 'No patterns found' ),
-			manage: __( 'Manage all template parts' ),
+			manage: __( 'Manage template parts' ),
```

Comment:
I like shorter, but since you're already seeing TPs in the library, is this actually clearer? Not a strong opinion, but why would I want to click "Manage template parts" if I can already see them?

Reply:
Hmm good point. 

I suppose it depends how you read it. For me the onus is on 'manage' rather than 'all'. So to answer your question: you'd click if you want to _manage_ what you can see. IE quickly delete / rename a bunch of template parts. From that perspective 'all' is kind of implied – you wouldn't jump directly into managing only a portion of template parts.

That said, I notice we currently have "Manage **all** templates" in the Templates panel 🙈 So perhaps we leave this one change out for now.